src/js/index.js

```diff
@@ -35,11 +37,9 @@ class FHCookieGuard {
         this._onAcceptCookiesClick = this._onAcceptCookiesClick.bind(this);
         this._onRefuseCookiesClick = this._onRefuseCookiesClick.bind(this);
         this.init();
+        this._initAutoAcceptCookieConsent();
```

Comment:
Deze kan dan eigenlijk ook wel public zijn, sowieso even bekijken wat we waar initialiseren.